### ISSUE TITLE
Add a proposal space for Parca

### ISSUE THREAD

**Issue by kakkoyun** [parca-dev/docs#159]
Add a page to list links to the Parca design documents or proposals

**Comment by kakkoyun** [comment]
Yes, exactly this, actually. Maybe an automated mechanism to just list whatever we have in the public directory. I think we just need to make it more visible.

**Comment by brancz** [comment]
Works for me.
